boards/arm/nucleo_f446ze/nucleo_f446ze.dts

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2018 Philémon Jaermann
+ * Copyright (c) 2019 Peter Nimac
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f4/stm32f446Xe.dtsi>
+#include "arduino_r3_connector.dtsi"
+
+<<<<<<< HEAD
```

Comment:
Please resolve merge conflicts.

Reply:
Could you please help me out a bit?
I've checked all of the files and on my local machine git status returns:
"nothing to commit, working tree clean"

Shippable still fails some checks and prints in the console to resolve merge conflicts.

Sorry, I know this may be a silly request but I've never really worked with git before.

Reply:
> Sorry, I know this may be a silly request but I've never really worked with git before.

@ArtifixPrime, that's ok, there's a first time for everything. I won't be able to help today, but Iwill come back to you this week.

Reply:
Alright, thank you so much! 
Sorry for replying this late, I am also busy with my own work, so whenever you'll find the time